static/js/components/Embedly.js

```diff
@@ -0,0 +1,55 @@
+// @flow
+import React from "react"
+
+export default class Embedly extends React.Component<*> {
+  renderEmbed() {
+    const { embedly } = this.props
+
+    // the response includes HTML which can be used to load a rich embed
+    // (either a video or the 'rich' type)
+    if (embedly.html) {
+      return (
+        <div
+          className="embed-container"
+          dangerouslySetInnerHTML={{ __html: embedly.html }}
+        />
+      )
+    }
+
+    if (embedly.type === "photo") {
+      return (
+        <div className="photo">
+          <img src={embedly.url} />
+        </div>
+      )
+    }
+
+    return (
```

Comment:
We should handle urls that Embedly considers invalid (e.g. ones it can't resolve). Any easy test case for this is a url to a local OD post.

Reply:
I added something so that if we get an error response back from embedly we just don't show anything. Curious if @roberthouse54 you have any ideas of what we should do in this case (that a user adds a post with a link but Embedly, for whatever reason, doesn't return any embed information for the link)